components/eventing-controller/pkg/backend/nats/jetstream/jetstream_test.go

```diff
@@ -43,59 +48,77 @@ type jetStreamClient struct {
 // created when no stream exists in JetStream.
 func TestJetStreamInitialize_NoStreamExists(t *testing.T) {
```

Comment:
should we move these common tests into a separate file? so that later when we depreciate code we do not delete these as well.

Reply:
Actually all the tests are being reworked on. I would say its ok for now. I will add a comment if needed

Reply:
Thinking about it, I think its a good idea to do it! Done